core/adapters/multiply_perform_sgx.go

```diff
@@ -2,47 +2,16 @@
 
 package adapters
 
-/*
```

Comment:
Not sure why you removed this?

Reply:
Yes, that makes absolutely no sense.  Fixing.